components/Aside.vue

```diff
@@ -1,10 +1,26 @@
 <template>
 	<aside>
 		<ul>
-			<li v-for="h in headers" :key="h.id">
+			<!-- <li v-for="h in headers" :key="h.id">
 				<nuxt-link :to="h.href">
 					{{ h.title }}
 				</nuxt-link>
+			</li> -->
+			<!-- The above <li> below are hard coded to show an example of how styles will look. This will be replaced by the commented out html code above. -->
```

Comment:
I think there is a mistake in the comment. I think "above" shouldn't be at the beginning of the sentence.

Reply:
Maybe make this a TODO as well.